test_six.py

```diff
@@ -1050,3 +1050,8 @@ def test_ensure_text(self):
             assert converted_unicode == self.UNICODE_EMOJI and isinstance(converted_unicode, str)
             # PY3: bytes -> str
             assert converted_binary == self.UNICODE_EMOJI and isinstance(converted_unicode, str)
+
+
+def test_foo():
+    foo = "FOO"
+    assert foo == "FOO"
```

Comment:
I believe this was added for debug purposes and should be removed from the final version.
```suggestion
```